### DISCUSSION TITLE
Create|Edit modals on the filament page

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#1641] [Help]
How can I put a create modal on the filament page?
The menu items section is completely livewire component on the filament page as well as every item. Each item must also be updated with a modal.

![9jf06VN](https://user-images.githubusercontent.com/29732308/154366014-131770cd-6efa-47b4-b800-9291237faeb9.png)

**Comment by danharrin** [comment]
How about adding it as an action to the page? You can make it open a modal with a create form there :)

https://filamentphp.com/docs/2.x/admin/pages#actions

**Reply by studiowizjo** [reply]
Here is example: https://github.com/laravel-filament/filament/discussions/1425#discussioncomment-2204036
